src/core/interaction/trackball.cpp

```diff
@@ -157,7 +157,8 @@ Trackball::Trackball(TrackballObject* object)
 
     auto systemSettings = InviwoApplication::getPtr()->getSettingsByType<SystemSettings>();
     followObjectDuringRotation_ = systemSettings->followObjectDuringRotation_.get();
-    systemSettings->followObjectDuringRotation_.onChange([systemSettings,this]() {
+    systemSettings->followObjectDuringRotation_.onChange([this]() {
```

Comment:
What was wrong with the old code?

Reply:
Nothing wrong with this one but the other one took the system settings pointer by reference. I think that it is not too good to store a pointer either so I changed to get it each time.